modules/creating-an-aws-role.adoc

```diff
@@ -0,0 +1,107 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-06-24
+:_mod-docs-content-type: PROCEDURE
+
+[id="creating-an-aws-role_{context}"]
+= Creating an AWS IAM role
+
+Create an {aws-first} IAM role that your service account can assume to securely access AWS resources.
+
+The following procedure demonstrates creating an {aws-short} IAM role by using the {aws-short} CLI. You can alternatively use the Cloud Credential Operator (CCO) utility `ccoctl`. Using the `ccoctl` utility creates many fields in the IAM role policy that are not required by the `ClusterLogForwarder` custom resource (CR). These extra fields are ignored by the CR. However, the `ccoctl` utility provides a convenient way for configuring IAM roles. For more information see link:https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html/authentication_and_authorization/managing-cloud-provider-credentials#cco-short-term-creds[Manual mode with short-term credentials for components].
+
+.Prerequisites
+
+* An {product-title} cluster with {sts-first} enabled and configured for {aws-short}.
+* Administrator access to the {aws-short} account.
+* {aws-short} CLI installed. 
+
+.Procedure
+
+. Create an IAM policy that grants permissions to write logs to CloudWatch.
+
+.. Create a file, for example `cw-iam-role-policy.json`, with the following content:
++
+[source,json]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "logs:PutLogEvents",
+                "logs:CreateLogGroup",
+                "logs:PutRetentionPolicy",
+                "logs:CreateLogStream",
+                "logs:DescribeLogGroups",
+                "logs:DescribeLogStreams"
+            ],
+            "Resource": "arn:aws:logs:*:*:*"
+        }
+    ]
+}
+----
+
+.. Create the IAM policy based on the policy definition above, by running the following command:
++
+[source,terminal]
+----
+aws iam create-policy \
+    --policy-name cluster-logging-allow \
+    --policy-document file://cw-iam-role-policy.json
+----
++
+Note down the `Arn` value of the created policy.
+
+. Create an IAM role for your {product-title} service account to assume.
```

Comment:
This step should be "create trust-policy", rather than create role.
ex. "Create a trust policy to allow the logging service account to assume an IAM role"

Reply:
done